README.md

```diff
@@ -209,32 +210,68 @@ Tree2
 * The resulting tag set used for coloring the track scheme branch
   view. ![Trackscheme Branch View with tags](doc/deep_lineage/lineage_classification/trackscheme_branch_with_tags.png)
 
-## Label image exporter
+## Exports
+
+### Label Image Exporter
 
 * The Label image exporter is capable of saving a label image to a file using the existing ellipsoids in Mastodon.
 * For the labels, the _spot ids_, _branch spot ids_ or the _track ids_ that correspond to the spots / ellipsoids may be
   used. Since these Ids are counted zero based in Mastodon, an **offset of 1** is added to all Ids so that no label
   clashes with the background of zero.
 * The recommended export format is '*.tif'-files. However, it should work also for other formats supported by ImageJ.
 * The export uses an image with signed integer value space, thus the maximum allowed id is 2.147.483.646.
-* Exporter can be found here:  ![Plugin Export Menu](doc/deep_lineage/export_label_image/plugin_export_menu.png)
-* The dialog:  ![Plugin Export Dialog](doc/deep_lineage/export_label_image/plugin_export_dialog.png)
+* Exporter can be found here:  ![Plugin Export Menu](doc/deep_lineage/export/label_image/plugin_export_menu.png)
+* The dialog:  ![Plugin Export Dialog](doc/deep_lineage/export/label_image/plugin_export_dialog.png)
 
-### Parameters:
+#### Parameters
   * Label Id: The id that is used for the labels. The default is the Spot track Id.
     * The ids correspond to the highlighted columns in the feature
-      table: ![Feature Table](doc/deep_lineage/export_label_image/plugin_export_table.png)
+      table: ![Feature Table](doc/deep_lineage/export/label_image/plugin_export_table.png)
   * Frame rate reduction: Only export every n-th frame. 1 means no reduction. Value must be >= 1.
     * The frame number corresponds to the _Spot frame_ column in the feature table.
 * Save to: Path to the file to save the label image to. Should end with '.tif'.
 
-### Example:
+#### Example
 
 * Demo data: [Example data set](https://github.com/mastodon-sc/mastodon-example-data/tree/master/tgmm-mini)
 * The timelapse with the ellipsoids in
-  BigDataViewer: ![BigDataViewer](doc/deep_lineage/export_label_image/bdv_timelapse.gif)
+  BigDataViewer: ![BigDataViewer](doc/deep_lineage/export/label_image/bdv_timelapse.gif)
 * The exported tif imported into [Napari](https://napari.org/stable/) 3D
-  view: ![Napari](doc/deep_lineage/export_label_image/napari_timelapse.gif)
+  view: ![Napari](doc/deep_lineage/export/label_image/napari_timelapse.gif)
+
+### GraphML Exporter
+
+* Exports the branch graph to a [GraphML](http://graphml.graphdrawing.org/) file.
+  * The graph is directed. The branch spots are the vertices and the branch links are the edges.
+  * The vertices receive a label attribute with the branch spot name. The vertices receive a duration attribute with the
+    branch duration.
+  * The edges are not labeled. The edges are not attributed.
```

Comment:
Suggestion: The edges are not labeled and have no attributes.

Reply:
Yes, this is better. I changed it.